frontend/src/pages/Users/Users.js

```diff
@@ -41,7 +42,7 @@ const Users = props => {
   const onClick = () => (tabIndex === 0 ? showDashboardDialog("addUser") : showDashboardDialog("addGroup"));
   const permissionIconDisplayed = allowedIntents.includes("global.listPermissions");
   return (
-    <div id="userdashboard" style={styles.container}>
+    <div id="userdashboard" data-test="userdashboard" style={styles.container}>
```

Comment:
do we still need the id "userdashboard" here?

Reply:
no, removed